contracts/validator-registration-contract/deployVRC/deployVRC.go

```diff
@@ -136,10 +146,46 @@ func main() {
 		log.WithFields(logrus.Fields{
 			"address": addr.Hex(),
 		}).Info("New contract deployed")
+
+		if k8sConfigMapName != "" {
+			if err := updateKubernetesConfigMap(k8sConfigMapName, addr.Hex()); err != nil {
+				log.Errorf("Failed to update kubernetes config map: %v", err)
```

Comment:
Actually, I think this should be a fatal condition. 

I’d like this process to fail if it tried to update k8s with an error.